### PR TITLE
Fixed the perldoc for MRIProcessingUtility following PR #275

### PR DESCRIPTION
**Pull request by cmadjar**
Also noticed that a variable was not used anymore in a function so I removed it from the function and the calls to the function.

### CODE REVIEW

File: docs/scripts_md/MRIProcessingUtility.md [hunk 1]
```diff
@@ -261,13 +277,17 @@ Register programs.
...
 
-Moves and updates the tarchive table with the new location of the tarchive.
+Moves and updates the `tarchive` table with the new location of the
+`tarchive`.
 
-INPUTS: tarchive location, tarchive information hashref
+INPUTS:
+  - $tarchive\_location: `tarchive location`,
+  - $tarchiveInfo     : `tarchive` information hashref,
+  - $upload\_id        : upload ID of the study
```
Comment (MounaSafiHarab) [inline docs/scripts_md/MRIProcessingUtility.md]: not important but I know you care about these,  so just wanted to let you know, but ignore if you wish.
not sure why the `:` are not aligned (same for some functions below this as well).

Comment (cmadjar) [reply docs/scripts_md/MRIProcessingUtility.md]: weird. It looks like that only in the md code. However, the MD file in the browser looks OK (it always ignores spaces anyway). I checked the .pm and all looked aligned so should be OK. Thanks for checking, my inner OCD thanks you ;).

**Comment by MounaSafiHarab** [comment]
@cmadjar 
reviewed it and all is good (I even counted that in PR #275, 16 functions had to be changed and I did find 16 here changed as well in MRIProcessingUtility.pm)

@nicolasbrossard 
since the changes are a bit more elaborate than the the other 2 I merged today, do you think you can look at this one, perhaps pass manual test it? and if all good, merge it?

**Comment by nicolasbrossard** [comment]
I noticed that when documenting methods that take a lot (>5) of parameters, the first few are named and then '...' is added at the end of the parameter list. I think we should always list the complete set of parameters but that's just my preference. I leave it to you to decide whether you want to change that or not.

**Comment by MounaSafiHarab** [comment]
> I noticed that when documenting methods that take a lot (>5) of parameters, the first few are named and then '...' is added at the end of the parameter list. I think we should always list the complete set of parameters but that's just my preference. I leave it to you to decide whether you want to change that or not.

indeed, and this is why counting the 16 functions took me forever, as 3 of those had the `(...`) in them and I could not find them!
But I think I asked @cmadjar that question in the past and I believe she was saying it is automatically done this way, non?

**Pull request merged**